### PR TITLE
Use florianl/go-conntrack library to retrieve conntrack table data

### PR DESCRIPTION
**Pull request by gbraxton**
Use of library prevents omission of connection entries do to flaws in the script regex and is much more cleaner.

### CODE REVIEW

**Comment by gbraxton** [comment]
resolves #6

**Review by tremaineeto — APPROVED** [review]
Looks good to me @gbraxton.

File: internal/agent/conntrack.go [hunk 1]
```diff
@@ -1,81 +1,59 @@
...
 
 	"net/http"
-	"os"
-	"os/exec"
-	"strconv"
-	"strings"
 	"time"
 
 	"github.com/golang/glog"
 )
```
Comment (bhaskar-reddy) [inline internal/agent/conntrack.go]: Good to see the reference to the TCP ENUMs 👍

**Review by bhaskar-reddy — APPROVED** [review]
Looks good to me @gbraxton !

**Pull request merged**